source/connect/tls-ssl-connection.txt

```diff
@@ -11,3 +11,86 @@ TLS/SSL Connection Tab
    :backlinks: none
    :depth: 1
    :class: singlecol
+
+The :guilabel:`TLS / SSL` tab allows you to connect deployments via TLS / SSL.
+For more information on :guilabel:`TLS / SSL`, see :manual:`TLS Options 
+</reference/connection-string/#tls-options>`
+
+Procedure
+---------
+
+.. procedure::
+   :style: normal
+
+   .. step:: Click :guilabel:`Advanced Connection Options`.
+
+      .. figure:: /images/compass/advanced-connection-options.png
+         :figwidth: 690px
+         :alt: New Advanced Connection Options
+
+   .. step:: Click the :guilabel:`TLS / SSL` tab.
+
+      You have the option to use a ``Default`` connection or to set the TLS / SSL 
+      connection ``On`` or ``Off``. 
+
+      Default
+      ~~~~~~~
+
+      The ``Default`` option does not use the ``TLS / SSL`` connection option. 
+      You can specify additional options when using ``Default``. 
+      
+      TLS / SSL
+      ~~~~~~~~~
+
+      The ``On`` option is enabled when using a :manual:`DNS seedlist (SRV) 
+      </reference/connection-string/#std-label-connections-dns-seedlist>` in 
+      the connection string. You can specify additional options when using 
+      ``On``.
```

Comment:
Consider flipping this sentence so the condition is first. I think we can also be a bit more specific as to what these options are.

When SSL/TLS Connection is ``On``, you can specify additional certificate options for your connection string.

Reply:
Done.